templates/Gemfile.local.rb.erb

```diff
@@ -0,0 +1,4 @@
+group :development do
+  gem 'pry-byebug'
+  gem 'puma'
```

Comment:
I'm a bit hesitant to force everyone to use puma given. Unlike byebug which is pretty much the only gem for debugging ruby 2.0+, there are many webservers for rails.